index.html

```diff
@@ -0,0 +1,82 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <title>Portfolio</title>
+</head>
+<body>
+  <section class="intro">
+
+<div class="logo">
+    <h1>Eros</h1>
+</div>
+  <div class="navbar">
+   <ul>
+       
+        <li><a href="Home">Home</a></li>
+        <li><a href="Porfolio">Portfolio</a></li>
+        <li><a href="Contact">Contact</a></li>
+
+    </ul>
+  </div>
+</section>
```

Comment:
- [ ] Please kindly use the `header` tag instead of the `section` tag for your project header for semantic reasons. :smile: 
- [ ] Please make sure you use the `a` tag for your logo instead of `h1` since the `h1` tag should be used for headings in your project. Besides, it will help you navigate your project easily by clicking on the logo to get back to your homepage. :+1: 
- [ ] Please from the project requirements, you should stick to the [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=37%3A1288) design as much as possible. But as you can see you have some differences in the design. You can see the design below and compare it with your project design. :pray:

| Your Design(No hambugger) |Figma Design (With hambugger)|
| :---:   | :---: | 
| ![Screenshot from 2022-10-31 11-14-38](https://user-images.githubusercontent.com/92303891/198996484-df17ca7a-12ef-4e21-9d16-0980332db5f5.png) | ![Screenshot from 2022-10-31 11-21-00](https://user-images.githubusercontent.com/92303891/198996735-e0c47379-93c5-429b-b855-800cea83b1ed.png)|